app/assets/javascripts/discourse/app/components/user-nav.hbs

```diff
@@ -0,0 +1,100 @@
+<section class="user-primary-navigation">
+  <ul class="main-nav nav nav-pills user-nav">
+    {{#unless @user.profile_hidden}}
+      <li class="summary">
+        <LinkTo @route="user.summary">
+          {{d-icon "user"}}
+          {{i18n "user.summary.title"}}
+        </LinkTo>
+      </li>
+
+      <UserNav::DropdownList
+        @icon="stream"
+        @text={{i18n "user.activity_stream"}}
+        @isActive={{eq @currentParentRoute "userActivity"}}
+        @class="user-activity" >
+
+        <:submenu>
```

Comment:
Do you mind giving a quick explanation of what this does? It seems like it's a kind of `yield` but I can't tell if it's an ember thing or a component or something else

Reply:
This is a [named block](https://api.emberjs.com/ember/release/modules/@glimmer%2Fcomponent)